media/css/firefox/whatsnew/includes/_header-deprecated.scss

```diff
@@ -0,0 +1,62 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+// * -------------------------------------------------------------------------- */
+// Up-to-date page header
+
+.c-page-header {
+    background: transparent;
+
+    .mzp-l-content {
+        padding-top: $spacing-lg;
+        padding-bottom: $spacing-lg;
+    }
+}
+
+.mzp-c-notification-bar {
+    display: none;
+    margin: $layout-xs auto;
+}
+
+.is-up-to-date .mzp-c-notification-bar.up-to-date,
+.is-out-of-date .mzp-c-notification-bar.out-of-date {
+    display: block;
+}
+
+.c-page-header-inner {
+    align-items: center;
+    display: grid;
+    grid-template-columns: 1fr;
+}
+
+.c-page-header-up-to-date {
+    margin: 0 $layout-sm;
+}
+
+.c-page-header-logo-fx {
+    @include image-replaced;
+    background: url($image-path + '/logos/firefox/browser/logo-word-hor.svg') no-repeat;
+    @include background-size(216px, 40px);
+    width: 216px;
+    height: 40px;
+    margin: 0;
+
+    @media (prefers-color-scheme: dark) {
```

Comment:
Same idea as above here, change old WNP header style partials to use header-deprecated and delete once they are outdated. 

The other option is to move this header block into the old WNP individual stylesheets and not have a deprecated partial import.